README.md

```diff
@@ -31,8 +42,24 @@ as provided by [Frab][frab-github]. The file format produced by the predecessor
 * In general it is possible to re-deployed the app for other events which
 provide the same event file format.
 
+## Build instructions
+
+Build instructions for configuring your build variants (i.e. debug and release).
+
+### Debug build / signing
+
+Before importing the project in Android Studio
+
+1. Create a file `gradle.properties` under directory `app`.
+2. Add the following JVM arguments in the file file `app/gradle.properties`.
+    ```
+    # Specifies the JVM arguments used for the daemon process.
+    # The setting is particularly useful for tweaking memory settings.
+    org.gradle.jvmargs=-Xmx2048m -Xms512m -XX:MaxPermSize=768m
+    ```
+3. Copy the signing configuration from the file [`app/gradle.properties.example`](app/gradle.properties.example) to `app/gradle.properties`.
```

Comment:
Signing should not be needed for the `debug` flavor, only for release builds.

Reply:
I was getting errors without having the signing during the debug. As gradle was complaning about the signing when running the app. I could only overcome this by commenting out the signing config and the line that referes to signing in the product flavours.

![Screen Shot 2019-05-13 at 22 41 38](https://user-images.githubusercontent.com/845598/57652781-9aaa2500-75d0-11e9-9bf7-d4e09593dfe1.png)

Is there any other way to do this?

Reply:
I am sorry for the misleading setup. You [won't need](https://developer.android.com/studio/publish/app-signing#debug-mode) to manually setup the signing configuration for `debug` at all. In order to "guide" a new developer I break the build and explain in the [error message](https://github.com/EventFahrplan/EventFahrplan/blob/master/app/build.gradle#L13-L21) that the `gradle.properties` must be created. There is a `gradle.properties.example` as a template. Maybe this is not clear enough.

Reply:
Okay, I understand now. I think both debug and release steps have to combined as a single step to tell the user to 
- Create a file under `app` directory and name it as `gradle.properties`.
- Then copy the contents from the file `gradle.properties.example` and pasting it into the `gradle.properties`.
Which is nothing but copying the signing configurations.

Having a explicit step mentioning the build type`release build / signing` makes it confusing. as you expect the debug build type is something different but in this case it is not.

what do you think?

Reply:
Sounds reasonable for the moment. Feel free to change the instructions.
In the long one could look into rewriting the build script in order to only configure signing for the release builds. But that's another issue.

Reply:
I made the changes in this commit 
af8c7cb